requirements.txt

```diff
@@ -1,3 +1,3 @@
-python-dateutil
-python-dotenv
-requests
+platformdirs==3.8.0
```

Comment:
I like to always specify the version in requirement files so that we have control when we update dependencies